man/man8/zpool.8

```diff
@@ -2430,7 +2439,6 @@ See
 .Xr zpool-features 5
 for details on compatibility with systems that support feature flags, but do not
 support all features enabled on the pool.
-.Bl -tag -width Ds
```

Comment:
This removal (and the .El below) are wrong, as it then unindents the -a and -V. Bring them both back and this will be formatted correctly.

Put differently, relative to what you have here, apply this diff:
```
--- zpool.8.orig	2019-04-14 13:46:43.408967141 -0500
+++ zpool.8	2019-04-14 13:46:24.900746044 -0500
@@ -2439,6 +2439,7 @@
 .Xr zpool-features 5
 for details on compatibility with systems that support feature flags, but do not
 support all features enabled on the pool.
+.Bl -tag -width Ds
 .It Fl a
 Enables all supported features on all pools.
 .It Fl V Ar version
@@ -2448,6 +2449,7 @@
 flag is specified, no features will be enabled on the pool.
 This option can only be used to increase the version number up to the last
 supported legacy version number.
+.El
 .It Xo
 .Nm
 .Cm version
```

Reply:
@rlaager Can you please check zfs.8 as well to make sure it is sane?

Reply:
zpool.8 done

Reply:
They both look good.